tools/config/go-templates/api.mustache

```diff
@@ -59,6 +59,16 @@ type {{#structPrefix}}{{&classname}}{{/structPrefix}}{{^structPrefix}}Api{{/stru
 {{/allParams}}
 }
 
+{{#vendorExtensions.x-xgen-paginated}}
+func (r {{#structPrefix}}{{&classname}}{{/structPrefix}}{{^structPrefix}}Api{{/structPrefix}}{{operationId}}Request) ListOptions(listOptions *ListOptions) {{#structPrefix}}{{&classname}}{{/structPrefix}}{{^structPrefix}}Api{{/structPrefix}}{{operationId}}Request {
+	r.pageNum = &listOptions.PageNum
```

Comment:
These are hard-coded for this POC, but can auto-generated in the future based on other extensions